presto-hive/src/main/java/com/facebook/presto/hive/CachingDirectoryLister.java

```diff
@@ -156,4 +154,29 @@ public long getRequestCount()
     {
         return cache.stats().requestCount();
     }
+
+    private static class CachedTableChecker
+    {
+        private final Set<SchemaTableName> cachedTableNames;
+        private final boolean cacheAllTables;
+
+        public CachedTableChecker(List<String> cachedTables)
+        {
+            this.cacheAllTables = cachedTables.contains("*");
```

Comment:
s/this.cacheAllTables/cacheAllTables/g
I think we prefer not to use this to reference variables